packages/nextjs/components/placeBid.tsx

```diff
@@ -1,18 +1,27 @@
+import { Bee } from "@ethersphere/bee-js";
 import { ContractFunctionExecutionError } from "viem";
 
 export const placeBid = async (
+  // Shoud do a data array
   project_id: string,
-  externalDescription: string,
+  description: string,
   timeSpan: number,
   price: bigint,
   writeAsync: any,
   setBidIsBidded: React.Dispatch<React.SetStateAction<boolean>>,
+  storage: Bee | undefined,
 ) => {
+  const writeProjectDetailsToStorage = async function () {
+    const res = await storage?.uploadData(
+      "f1e4ff753ea1cb923269ed0cda909d13a10d624719edf261e196584e9e764e50",
+      JSON.stringify({ project_id: project_id, description: description, price: Number(price), timeSpan: timeSpan }),
```

Comment:
We should use other json format here,  it is bid ,  not project,  let's rename `project_id` to `bid_id` at least